java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -367,6 +373,10 @@ protected String handleStringOrNullResponse(Response response) throws RedisExcep
                 String.class, EnumSet.of(ResponseFlags.IS_NULLABLE, ResponseFlags.ENCODING_UTF8), response);
     }
 
+    protected GlideString handleBytesResponse(Response response) throws RedisException {
```

Comment:
But you need `handleBytesResponse` which returns `byte[]`. Align with @yipin-chen and #1621